Main/gyro.ino

```diff
@@ -117,18 +117,18 @@ void outputSensorValues() {
 
 void detectRotation() {
 
-  // Need these methods to populate ypr
+  //Need these methods to populate ypr
   mpu.dmpGetQuaternion(&q, fifoBuffer);
+  mpu.dmpGetAccel(&aa, fifoBuffer);
   mpu.dmpGetGravity(&gravity, &q);
   mpu.dmpGetYawPitchRoll(ypr, &q, &gravity);
 
   int yawNumber = ypr[0] * 180/M_PI; // get value of yaw number out of ypr array and convert to radians
-
+  yawNumber=removeNegativeSign(yawNumber);
   printIntToSerial(yawNumber); // use a pretty print function from helpers.ino
-
```

Comment:
we used the exact yaw number as the brightness for the ight here, maybe we could try mapping it so instead of a value between 0 and 180 its a value between 0 and 255